var/spack/packages/spot/package.py

```diff
@@ -0,0 +1,18 @@
+from spack import *
+import os
+
+class Spot(Package):
+    """Spot is a C++11 library for omega-automata manipulation and model checking."""
+    homepage = "https://spot.lrde.epita.fr/index.html"
+    url      = "http://www.lrde.epita.fr/dload/spot/spot-1.99.3.tar.gz"
+
+    version('1.99.3', 'd53adcb2d0fe7c69f45d4e595a58254e')
+
+    #depends_on("gcc@4.8:")
```

Comment:
Good to know this has been discussed before. Yes, we require something like the following:

```
require('%gcc@4.4.7:')
```

One reason we require this feature is because specific Boost library versions are only tested with certain GCC compiler versions, and the ROSE support matrix is based on this Boost-GCC support matrix. For example:

```
depends_on('boost@1.54.0', when '%gcc@4.4.7:4.7.3')
depends_on('boost@1.55.0', when '%gcc@4.8.1')
```